tests/framework/framework.go

```diff
@@ -462,7 +462,7 @@ func (f *Framework) CreateQuotaInSpecifiedNs(ns string, requestCPU, requestMemor
 		if err != nil {
 			return false, err
 		}
-		return len(quota.Status.Hard) == 4, nil
+		return len(quota.Status.Used) == 4, nil
```

Comment:
According to https://pkg.go.dev/k8s.io/api/core/v1#ResourceQuotaStatus, 
```go
	// Used is the current observed total usage of the resource in the namespace.
```
How do we expect this value to reach 4 if we are just "getting" the quota in the PollImmediate and not creating/doing anything? I think I'm missing something because the same reasoning could be applied for `Hard`

Reply:
And how do we expect `Used` to already be 4 when the quota has just been created? It made sense with `Hard` since:
```go
	// Hard is the set of enforced hard limits for each named resource.
```

Reply:
So `used` keys take a couple of seconds to be created, that's why we need to wait for that
If you follow the k8s links in the PR description, that is how the admission hook works as well,
it would only proceed if the `used` keys were created. Note we are not waiting for the usage to go up or anything, we just need the keys that track the usage to be there